crds/examples/AzureApplicationGatewayLoadDistributionPolicy.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: appgw.ingress.azure.io/v1beta1
+kind: AzureApplicationGatewayLoadDistribitionPolicy
+metadata:
+  name: load-distribution-policy-with-failover
+spec:
+  targets:
+    - backend:
+        service:
+          name: service-1
+          port:
+            number: 80
+      role: active
+      weight: 8
+    - backend:
+        service:
+          name: service-2
+          port:
+            number: 80
+      role: active
```

Comment:
passive?

Reply:
correct, this should be passive to be more in line with example. will fix